### ISSUE TITLE
multi-threaded HttpTest mixed normal FlurlClient requests

### ISSUE THREAD

**Issue by engenb** [tmenier/Flurl#104]
Hello,

Not sure this is something you're interested in supporting, but thought I'd report an issue I'm running into.

My test project has a mixture of unit tests that use HttpTest and some integration tests that are actually hitting a remote api and verifying the expected response is returned.

If I run my tests in parallel, this causes problems due to HttpTest altering the static FlurlHttp config - which is shared by both HttpTest and normal, non-test FlurlClient.

Also with tests running in parallel, I'm having a mock response defined in one HttpTest being returned for a response in another test non-HttpTest.  I haven't dug into this too far as I'm assuming it's something related to the above issue.

**Comment by tmenier** [comment]
Duplicate of #67
